filebeat/tests/system/test_crawler.py

```diff
@@ -48,6 +48,9 @@ def test_fetched_lines(self):
         # Check that output file has the same number of lines as the log file
         assert iterations == len(output)
 
+        # Check that the application did not panic
+        assert self.did_not_panic()
+
```

Comment:
Agree, we should definitively move both to tear down methods in TestCase. I'm sure there is also an annotation to skip TearDown for exceptions where for example exit should be 1 and not 0 (which is probably default)